lib/hostel/site.rb

```diff
@@ -22,7 +22,7 @@ def build_path(path, secure: true, path_args: {})
         'http://'
       end
 
-      url << domain
+      url << ENV['QA_HOST']?ENV['QA_HOST']:domain
```

Comment:
nitpick but can you add spacing here so it is a little more readable? like this
 `ENV['QA_HOST'] ? ENV['QA_HOST'] : domain`